onnxruntime/core/protobuf/onnx-ml.proto3

```diff
@@ -277,6 +287,9 @@ message GraphProto {
   // MAY also appear in the input list.
   repeated TensorProto initializer = 5;
```

Comment:
>initializer [](start = 23, length = 11)

Check for places in the code that deal with initializer and see if we need to add handling for sparse initializer